assets/responce/local.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think since this file is related only to spec folder it's better to place it somewhere like

`spec/assets/responce_example.xml`